src/framework/input/element-input.js

```diff
@@ -369,9 +369,13 @@ class ElementInput {
         this._touchmoveHandler = this._handleTouchMove.bind(this);
         this._sortHandler = this._sortElements.bind(this);
 
+        /** @type {ElementComponent[]} */
         this._elements = [];
+        /** @type {ElementComponent|null} */
         this._hoveredElement = null;
+        /** @type {ElementComponent|null} */
         this._pressedElement = null;
+        /** @type {Record<number, {x: number, y: number, camera: CameraComponent, element: ElementComponent}>} */
```

Comment:
I think this would be the first appearance of `Record` in the source's JSDocs. I can't find any resources documenting that (e.g. here: https://jsdoc.app/tags-type.html).

Reply:
Yea, I should definitely use `Object<..., ...>`. I am working on it + some other changes (converting to style without `@typedef`)